src/Razor/src/Microsoft.VisualStudio.RazorExtension/Options/AdvancedOptionPage.cs

```diff
@@ -136,14 +136,24 @@ protected override void OnApply(PageApplyEventArgs e)
             _optionsStorage.Value.AutoInsertAttributeQuotes = _autoInsertAttributeQuotes.Value;
         }
 
+        if (_commitElementsWithSpace is not null)
+        {
+            _optionsStorage.Value.CommitElementsWithSpace = _commitElementsWithSpace.Value;
+        }
+
         if (_colorBackground is not null)
         {
             _optionsStorage.Value.ColorBackground = _colorBackground.Value;
         }
 
-        if (_commitElementsWithSpace is not null)
+        if (_codeBlockBraceOnNextLine is not null)
```

Comment:
The diff looks bad here, but what really happened was I moved `_commitElementsWithSpace` above, so the ordering was consistent everywhere in the file, then added the code for `_codeBlockBraceOnNextLine`